dashboard/lib/policies/lti.rb

```diff
@@ -59,6 +59,15 @@ def self.issuer(user)
     nil
   end
 
+  # Converts the `issuer` or `iss` LTI value to a string we would show to
+  # users of Code.org
+  # Example: 'https://schoology.schoology.com' -> 'Schoology'
+  def self.issuer_name(issuer)
+    return 'Canvas' if /canvas/.match?(issuer)
+    return 'Schoology' if /schoology/.match?(issuer)
```

Comment:
Minor nit: instead of using a regex, you could check against the exact issuer values in the `LMS_PLATFORMS` const

Reply:
We talked offline and decided to keep the regex for now because Canvas might have multiple issuers depending on the environment they are trying to integrate with us: https://canvas.instructure.com/doc/api/file.lti_dev_key_config.html